File: 프로그래머스/lv0/120846. 합성수 찾기/합성수 찾기.py
def solution(n):
    answer = []
    count = 0
    for i in range(2, n+1): 
        for j in range(1, i+1): # 2중 포문 이용
            if i % j == 0:
                answer.append(i)
        if answer.count(i) >= 3: # 총 약수의 개수가 3개 이상이면
            count += 1 # 카운트를 한다
    return count

File: 프로그래머스/lv0/120846. 합성수 찾기/합성수 찾기.swift
import Foundation

func solution(_ n:Int) -> Int {

    var answer: Int = 0

    for i in 1...n {
        var count: Int = 0
        for j in 1...i {
            if i % j == 0 {
                count += 1
            }
        }
        if count > 2 {
                answer += 1
            }
    }
    return answer
}
